src/states/elder/mod.rs

```diff
@@ -1737,6 +1689,40 @@ impl Approved for Elder {
 
         Ok(())
     }
+
+    // Checking members vote status and vote to remove those non-resposive nodes.
+    #[cfg(feature = "mock_parsec")]
+    fn check_voting_status(&mut self) {
```

Comment:
For consistency, I'd either rename this to `check_vote_status` or rename `Chain::check_vote_status` to `check_voting_status`.